include/core/tracer_reaction_model.h

```diff
@@ -598,14 +653,18 @@ class GaussianLevelsetTracerReactionPrefactor
             const double k =
               tracer_reaction_constant_bulk +
               delta_reaction_constant *
-                exp(-pow(levelset_vec[i], 2) / squared_thickness);
-            jacobian_vector[i] =
-              k * (tracer_reaction_order - 1.) *
-              pow(concentration_vec[i], tracer_reaction_order - 2.);
+                std::exp(-std::pow(levelset_vec[i], 2) / squared_thickness);
```

Comment:
These powers of two can be just the thing multiplied by itself, but this is super minor